spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -31,9 +39,35 @@ case class ColumnName(columnName: String, alias: Option[String] = None) extends
   override def selectedAs = alias.getOrElse(columnName)
   override def toString: String = columnName
 
+  def overwrite() = CollectionColumnName(columnName, alias, CollectionOverwrite)
```

Comment:
Why non-nullary? This method doesn't have a side-effect, parentheses are not needed.
If you remove parentheses, it will be possible to use it as `"columnName".overwrite` without warning